### DISCUSSION TITLE
Issue Forms should still be available when Editing existing issues

### DISCUSSION THREAD

**Discussion by poyribn** [community/community#5108] [Projects and Issues]
When reporting a new issue, it is done by an "Issue Form" if supported by the repository.

But when editing a posted issue, the interface is the "Raw" Markdown Editor instead of the Issue Form,
which is much less clean and encourages more errors.

Even when the "Raw" Markdown Editor should be an option,
the Issue Form should still be the Default format for editing existing issue entries.
How about if, instead of converting the Issue Form to "Raw Markdown" immediately,
it is instead done by a button that the repository owners can disable?

**Comment by daniel-evers** [comment]
Especially for more complex templates with e.g. drop-down boxes, this would definitely be very helpful in my opinion. It would also greatly assist people who are not very familiar with Markdown. Mistakes could be avoided more easily.
